src/current/_includes/releases/v22.2/v22.2.14.md

```diff
@@ -0,0 +1,61 @@
+- Added two new [changefeed metrics](../v22.2/monitor-and-debug-changefeeds.html#recommended-changefeed-metrics-to-track): `changefeed.checkpoint_progress`  is similar to `changefeed.max_behind_nanos` but supports [metrics labels](../v22.2/monitor-and-debug-changefeeds.html#using-changefeed-metrics-labels); `changefeed.aggregator_progress metric` tracks the progress of individual aggregators (the lowest timestamp for which all aggregators with the label have emitted all values they're responsible for). [#109745][#109745]
+
+<h3 id="v22-2-14-bug-fixes">Bug fixes</h3>
+
+- Fixed a potential deadlock when running [changefeeds](../v22.2/create-changefeed.html) with `end_time` option set. [#108076][#108076]
+- Added cancel checking to [index](../v22.2/indexes.html) constraint initialization code to allow queries to timeout during [query optimization](../v22.2/cost-based-optimizer.html) if analyzing predicates to constrain an index starts using too many resources. Example of setting a timeout using the [`statement_timeout` session setting](../v22.2/set-vars.html#statement-timeout): `SET statement_timeout='5.0s';`.  [#106950][#106950]
+- Fixed the [schema changer](../v22.2/online-schema-changes.html) [job](../v22.2/show-jobs.html) when [`CREATE AS`](../v22.2/create-table-as.html) sources from [`SHOW CREATE FUNCTION`](../v22.2/show-create.html#show-the-create-function-statement-for-a-function), for example: `CREATE TABLE t AS SELECT * FROM [SHOW CREATE FUNCTION f];`.  [#108167][#108167]
+- Previously, CockroachDB could encounter an internal error `unexpected non-zero bytes limit for txnKVStreamer` when evaluating [locality-optimized](../v22.2/cost-based-optimizer.html#locality-optimized-search-in-multi-region-clusters) lookup [joins](../v22.2/joins.html) in case it had to perform the remote regions' lookup. The bug was introduced in [v22.2](../releases/v22.2.html) and is now fixed. A temporary workaround can be accomplished without upgrading by setting the `streamer_enabled` [session variable](../v22.2/set-vars.html): `SET streamer_enabled = false;`. [#108252][#108252]
+- Fixed a spurious error `no data source matches prefix` that could occur during [planning for a query](../v22.2/cost-based-optimizer.html) with `DISTINCT ON` and [`ORDER BY ASC NULLS LAST`](../v22.2/order-by.html) or `ORDER BY DESC NULLS FIRST`. [#108302][#108302]
+- Previously, using [`IMPORT INTO`](../v22.2/import-into.html) for `DELIMITED DATA` or MySQL imports would error with `column ... does not exist` if it was importing into a [collated string](../v22.2/collate.html) column. This is now fixed. [#108287][#108287]
+- Fixed a bug in the index recommendations provided in the EXPLAIN output where [`ALTER INDEX ... VISIBLE`](../v22.2/alter-index.html#not-visible) index recommendations may suggest making the wrong index visible when there are multiple invisible indexes in a table. [#108647][#108647]
```

Comment:
```suggestion
- Fixed a bug in the index recommendations provided in the `EXPLAIN` output where [`ALTER INDEX ... VISIBLE`](../v22.2/alter-index.html#not-visible) index recommendations may suggest making the wrong index visible when there are multiple invisible indexes in a table. [#108647][#108647]
```

Reply:
formatted as code and added a link for `EXPLAIN`.